### ISSUE TITLE
Addon does not load in Blender 2.78

### ISSUE THREAD

**Issue by nureyev** [antoni4040/MultiEdit-Addon#2]
The addon does not load or display a panel in Blender 2.78b.

**Comment by antoni4040** [comment]
I'm currently using it with Blender 2.78b and it's working fine so far. Could you be a bit more specific about the problem?

**Comment by nureyev** [comment]
Thanks for the reply. I spent some time debugging this. I think there's a conflict with another addon, or I made an error installing another addon.

Short version is a possible conflict with MiraTools. I resolved it by deleting the MiraTools addon and installing MultiEdit from the .py (not the .ZIP).

I was getting numerous errors in the terminal of the type:
fake_module: addon missing 'bl_info' gives bad performance!: 'C:\\Users\\********\\AppData\\Roaming\\Blender Foundation\\Blender\\2.78\\scripts\\addons\\mi_wrap_master.py'

Since I need your addon more than MiraTools, I don't mind this solution.

By the way, this is a great feature. Thanks again.

**Comment by antoni4040** [comment]
I installed MiraTools and seems to be working fine with MultiEdit installed. Maybe you have an older version of MiraTools that's causing problems? Try installing the most recent version. Anyway, I'm glad you find it useful, have fun modelling!

**Comment by nureyev** [comment]
I tried the newest MiraTools as you suggested, which now does not throw as many errors. Instead I usually get something like:
<!> event has invalid window
Saved session recovery to 'C:\Users\*********\AppData\Local\Temp\quit.blend'
Error: Not freed memory blocks: 34, total unfreed memory 7.538506 MB 

I think there's a problem, but not with your addon. I get a small memory leak (0.000648 MB) when I remove MiraTools but that's from another addon. There's some sort of interaction occurring between something I installed or changed.

I'll document it in case someone else has a similar issue.

I have all the standard addons installed + Mouselook Navigation 1.06, Bool Tools 0.3.5, ANT Landscape 0.1.4, Run Script in PyConsole 1.0, Multi Object UV Editing 0.9.8.

Operating system is Windows 10 (x64). Blender 2.78b (x64). 

Thanks for your time.
